builder/assignmentEnvImageBuilder.go

```diff
@@ -177,7 +200,11 @@ func (assgnEnv *assignmentEnvironment) writeToDockerfile() error {
 	return nil
 }
 
-func (assgnEnv *assignmentEnvironment) build() error {
+// build builds a docker image for the given language using the ImageBuild function
```

Comment:
you can just say **// build a docker image ...**
Again implementation detail issue.

Reply:
Done